site/upgrading-to-1.0.md

```diff
@@ -0,0 +1,257 @@
+    chmod a+x ./fluxctl_030
+
+The next steps depend on whether you
+
+ * [Connect Flux to Weave Cloud](#if-you-are-connecting-to-weave-cloud); or,
+ * [You are running Flux standalone](#if-you-are-running-flux-in-standalone-mode) (i.e., you run both
+   the Flux daemon and the Flux service, rather than going through
+   Weave Cloud).
+
+## <a name="weavecloud">If you are connecting to Weave Cloud</a>
+
+Set the environment variable `FLUX_SERVICE_TOKEN` to be your service
+token (found in the Weave Cloud UI for the instance you are
+upgrading).
+
+Before making changes, get the config so that it can be consulted
+later:
+
+    ./fluxctl_030 get-config --fingerprint=md5 | tee old-config.yaml
+
+### Remove old Flux resources
+
+> *Important! If you have Flux resources committed to git*
+> 
+> The first thing to do is remove any manifests for running Flux that
+> you have stored in git. Otherwise, when the new Flux daemon runs it
+> will restore the old configuration.
+
+Run Weave Cloud’s launch generator to delete the resources:
+
+    kubectl delete -n kube-system -f \
+      https://cloud.weave.works/k8s/flux.yaml?flux-version=0.3.0”
+
+### Delete deployment keys
+
+It’s good practice to remove any unused deployment keys. If you’re
+using GitHub, look at the settings for the repository you were
+pointing Flux at, and delete the key Flux was using.
+
+To check you are removing the correct key, get the fingerprint of the
+key used by Flux with
+
+    ./fluxctl_030 get-config --fingerprint=md5
+
+### Configure and run the new flux resources
+
+> First, it is important to understand that Flux manages more of your
+> cluster resources now. It will automatically apply manifests that
+> appear in your config repo, either by creating or by updating them. In
+> other words, it tries to keep the cluster running whatever is
+> represented in the repo. (Though it doesn’t delete things, yet.)
+
+To run the new Flux with Weave Cloud: 
+
+ * Go to your instance settings (the cog icon) and click “Deploy Config”
+ * Enter the git URL, path and branch values from the saved config
+   (old-config.yaml)
+ * Run the `kubectl` command shown below the form.
+ * Wait for the 
+ * Follow the instructions below that to push the deploy key (this is
+   also a good opportunity to delete any old keys, if you didn’t do
+   that above).
+
+You should now be able to click the “Deploy” tab and see your running
+system (again), with the updated Flux daemon.
+
+## If you are running Flux in "standalone" mode
+
+Set the environment variable FLUX_URL to point to the Flux service you
+are running, as described in
+[the old deployment docs](https://github.com/weaveworks/flux/blob/0.3.0/site/using.md#fluxctl-setup). The
+particular URL will differ, depending on how you have told Kubernetes
+to expose the Flux service.
+
+Before making any changes, get the config so that it can be consulted later:
+
+    ./fluxctl_030 get-config --fingerprint=md5 | tee old-config.yaml
```

Comment:
md5 is hardcoded in flux. If a user uses  --fingerprint=SHA1, he will get the same result, which may be a bit confusing. Probably nitpicking.

Reply:
Old `fluxctl` should probably have checked that the value supplied is `md5`, yeah. But I have to supply some value here, otherwise the output is the whole public key.